packages/cli/package.json

```diff
@@ -46,26 +46,26 @@
   "dependencies": {
     "@axe-core/webdriverjs": "^4.4.5",
     "axe-core": "^4.4.3",
-    "chromedriver": "^105.0.1",
+    "chromedriver": "^106.0.1",
     "colors": "^1.4.0",
-    "commander": "^9.3.0",
+    "commander": "^9.4.1",
     "selenium-webdriver": "4.3.0"
```

Comment:
Will need to pin at `4.3.0` - 

https://github.com/dequelabs/axe-core-npm/issues/580 - breaking change introduced